main.go

```diff
@@ -136,10 +144,22 @@ func do(githubInfo *github.GithubInfo, mavenRepositoryInfo *maven.RepositoryInfo
 			log.Fatalf("%v\n", err)
 		}
 
-		branchName := "bump-it-up/" + dependency.PropertyName + "/" + lastVersion
+		if *oneBranchPerDependency {
+			branchName := "bump-it-up/" + dependency.PropertyName + "/" + lastVersion
 
-		if err := repo.PushAndCreatePR(branchName, prTitle, prDescription); err != nil {
-			log.Printf("%v\n", err)
+			if err := repo.PushAndCreatePR(branchName, prTitle, prDescription); err != nil {
```

Comment:
if we are with *oneBranchPerDependency you need to reset prDescription in the for loop.